Automattic/wp-calypso
# migrate `InlineHelp` to use `react-query`

Working draft, still a bit rough around the edges. Don't merge yet.

#### Changes proposed in this Pull Request

* `InlineHelp`: use `react-query` to fetch search results

#### Testing instructions

* Open the Inline Help Popover by clicking on the blue FAB on the bottom right (not on `/home/<your-site>`)
* Perform a search by entering any kind of term into the search field
* Make sure it works exactly like on prod
* Make sure the `Get Help` section on the home dashboard (`/home/<your-site>`) still works as well
